manifests/repo/apt.pp

```diff
@@ -7,12 +7,16 @@
 
   if($::mongodb::repo::ensure == 'present' or $::mongodb::repo::ensure == true) {
     apt::source { 'mongodb':
-      location    => $::mongodb::repo::location,
-      release     => 'dist',
-      repos       => '10gen',
-      key         => '492EAFE8CD016A07919F1D2B9ECBEC467F0CEB10',
-      key_server  => 'hkp://keyserver.ubuntu.com:80',
-      include_src => false,
+      location   => $::mongodb::repo::location,
+      release    => 'dist',
+      repos      => '10gen',
+      key        => {
+        'id'     => '492EAFE8CD016A07919F1D2B9ECBEC467F0CEB10',
+        'server' => 'hkp://keyserver.ubuntu.com:80',
+      },
+      include    => {
+        'src'    => false
+      }
     }
```

Comment:
Thanks. They're internally consistent (and pass puppetlint, at least according to syntastic). They're aligned differently than before because the longest parameter was include_src and that's gone now. At least, that's how I'm interpreting this. 

Do you need me to change the rocket alignments?